zerver/openapi/zulip.yaml

```diff
@@ -21825,13 +21816,6 @@ components:
                   Only present if message's content was edited.
 
                   The rendered HTML representation of `prev_content`.
-              prev_rendered_content_version:
-                type: integer
-                description: |
-                  Only present if message's content was edited.
-
-                  The Markdown processor version number for the message
-                  immediately prior to this edit event.
```

Comment:
One should include the `Changes` entry for removed fields in the description of the parent, `edit_history`, near `[disable-edit-history]: /help/disable-message-edit-history` just below.

And I guess we should also do that in the event block for consistency. Not as important in that case, as it has long been documented as to-be-ignored, but a client whose parser is looking for the field would crash, and so it is valuable for documentation to cover every API change clearly.

Reply:
Updated:

<details><summary>`edit_history` object</summary>
<img src="https://github.com/user-attachments/assets/aca5ca0c-e1dc-47f6-b9bc-5195bb0cae55">
</img>
</details> 

<details><summary>`update_message` event type</summary>
<img src="https://github.com/user-attachments/assets/ee857cb7-8d76-4d9e-83ba-2c9c82a5f87a">
</img>
</details>